validator.py

```diff
@@ -863,27 +839,20 @@ def verify_uniqueness(self):
                                 vals_found.append(val)
 
 
-    def scenes_with_transitions(self):
-        '''
-        Looks through the yaml file to make sure that every scene from 0 to n-1 has 
-        a transitions field
-        '''
-        data = copy.deepcopy(self.loaded_yaml)
-        scenes = data['scenes']
-        for i in range(0, len(scenes)-1):
-            if 'transitions' not in scenes[i]:
-                self.logger.log(LogLevel.WARN, "Key 'transitions' must be provided within all but the last entry in 'scenes' but is missing at scenes[" + str(i) + "]")
-                self.missing_keys += 1
-
-
     def scenes_with_state(self):
         '''
-        Looks through the yaml file to make sure that every scene from 1 to n-1 has 
+        Looks through the yaml file to make sure that every scene from 1 to n has 
```

Comment:
Really trivial, but can we change this to "every scene except the first"?

Reply:
Updated, good catch on line 872 about that one final index to id